index.js

```diff
@@ -0,0 +1,78 @@
+/* eslint-disable max-classes-per-file */
+/* eslint-disable no-use-before-define */
+
+import Book from './modules/Book.js';
+import { DateTime } from './modules/luxon.js';
+import display from './modules/display.js';
+
+class Library {
+  constructor() {
+    this.data = [];
+  }
+
+  addBook = (book) => {
+    this.data.push(book);
+    localStorage.setItem('library', JSON.stringify(this.data));
+    addToUI(book);
+  };
+
+  removeBook = (id) => {
+    const book = document.getElementById(id);
+    book.remove();
+    this.data = this.data.filter((bookObj) => bookObj.id !== id);
+    localStorage.setItem('library', JSON.stringify(this.data));
+  };
+}
+
+const library = new Library();
+
+const getInput = () => {
+  const title = document.getElementById('bookTitle');
+  const author = document.getElementById('bookAuthor');
+  const book = new Book(title.value, author.value);
+  title.value = '';
+  author.value = '';
+  return book;
+};
+
+const addToUI = (bookObj) => {
+  let colorClass = '';
+  if (library.data.indexOf(bookObj) % 2 !== 0) {
+    colorClass = 'light';
+  } else {
+    colorClass = 'dark';
+  }
+  const bookList = document.getElementById('book-list');
+  const book = document.createElement('li');
+  book.classList.add('book');
+  book.classList.add(colorClass);
+  book.setAttribute('id', bookObj.id);
+  book.innerHTML = `<p><span>"${bookObj.title}"</span> by ${bookObj.author}</p>`;
+  const deleteBtn = document.createElement('button');
+  deleteBtn.innerHTML = 'Remove';
+  deleteBtn.addEventListener('click', () => library.removeBook(bookObj.id));
+  book.appendChild(deleteBtn);
+  bookList.appendChild(book);
+};
```

Comment:
- [ ] Currently, your application doesn't function as it should. When I click the navigation links, nothing happens. I can't add a book, delete a book or navigate to other pages. Kindly consider this and make it work. I know you can do it. 👍 

![chrome-capture-2023-0-23](https://user-images.githubusercontent.com/77106962/214125802-22ce409c-755e-46ac-b03c-93fd7abc5a29.gif)

Reply:
On my end it works well without any issue. @Gambit142 let's chat on zoom so i can show it. thank you

Reply:
Hi @hakichris, I am willing to jump on a zoom call to check why it works on your end an not mine 👍

Reply:
check your slack, please.

Reply:
Thank you so much Francis. I was able to fix.